feature_engine/timeseries/forecasting/base_forecast_transformers.py

```diff
@@ -165,6 +177,18 @@ def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
         if self.missing_values == "raise":
             self._check_na_and_inf(X)
 
+        if self.group_by_variables:
```

Comment:
Do we need to add this functionality to feature-engine? Pandas will fail if the variables are not in the dataframe.